lightkurve/targetpixelfile.py

```diff
@@ -867,6 +937,151 @@ def cutout(self, center=None, size=5):
             newfits = fits.HDUList(hdus)
         return self.__class__(newfits)
 
+    def from_fits_images(images, position, size=(11, 11), extension=1,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [TargetPixelFile.from_fits_images is too complex (19)](https://app.codacy.com/app/barentsen/lightkurve/pullRequest?prid=4287604)